client/app/bundles/shared/components/SideNav.jsx

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
```

Comment:
We need to import react-router flowtype annotations to annotate the `routes` variable, and then `routes.map((route, index) => (` below.